README.md

```diff
@@ -12,7 +12,7 @@ To obtain a deobfuscated Minecraft jar, [`./gradlew mapNamedJar`](#mapNamedJar)
 
 ## Contributing
 
-Please remember that copying and pasting mappings from alternate projects under more restrictive licenses (such as MCP) is **completely forbidden** without explicit permission from the 
+Please remember that copying and pasting mappings from alternate projects under more restrictive licenses (such as MCP or Mojang's obfuscation maps) is **completely forbidden** without explicit permission from the 
```

Comment:
Should this be more clear that even using their names for insprisation is still forbidden?

Reply:
> without permission from the owners  of said mappings

Permission to use the names isn't enough. For example, if there was some MIT-licensed mapping project, we'd have permission to use the names, but not distribute under CC0. This sentence should be changed to

> without permission from the owners of said mappings **to distribute the names under the CC0 license**

Reply:
I added a thing about inspiration and the fix about CC0 in d02a054.